src/net/MSRP/MSRPSession.cs

```diff
@@ -0,0 +1,67 @@
+﻿//-----------------------------------------------------------------------------
+// Filename: SDPMessageMediaFormat.cs
+//
+// Description: Contains enums and helper classes for common definitions
+// and attributes used in SDP payloads.
+//
```

Comment:
The filename and description need to be updated.

Reply:
fixed